perceval/backends/core/github.py

```diff
@@ -751,7 +751,7 @@ def user(self, login):
 
         url_user = urijoin(self.base_url, 'users', login)
 
-        logging.info("Getting info for %s" % (url_user))
+        logger.info("Getting info for %s" % url_user)
```

Comment:
Can you change the level of this entry too? I think this a debug message, not an info one.